prepare/kotlin-compiler-internal-test-framework/build.gradle.kts

```diff
@@ -19,5 +19,9 @@ dependencies {
 
 publish()
 runtimeJar()
-sourcesJar()
-javadocJar()
\ No newline at end of file
+val sourcesJarTask = sourcesJar()
+javadocJar()
+
+sourcesJarTask {
```

Comment:
does this use task configuration avoidance?

Reply:
it looks so
https://github.com/jprinet/kotlin/blob/3ee8aabe4cbae7ddc5175e85a440a9ae42677465/buildSrc/src/main/kotlin/tasks.kt#L239

Reply:
[it isn't](https://ge.solutions-team.gradle.com/s/kbc552lnvsyqq/performance/configuration?details=code-unit-prepare/kotlin-compiler-internal-test-framework/build.gradle.kts!:prepare:kotlin-compiler-internal-test-framework&openScriptsAndPlugins=WyJjb2RlLXVuaXQtYnVpbGQuZ3JhZGxlLmt0cyIsImNvZGUtdW5pdC06YnVpbGRzcmMhYnVpbGQuZ3JhZGxlLmt0cyIsImNvZGUtdW5pdC1zZXR0aW5ncy5ncmFkbGUiLCJjb2RlLXVuaXQtZ3JhZGxlLXBsdWdpbi1jb21tb24tY29uZmlndXJhdGlvbiIsImNvZGUtdW5pdC1wcmVwYXJlL2tvdGxpbi1jb21waWxlci1pbnRlcm5hbC10ZXN0LWZyYW1ld29yay9idWlsZC5ncmFkbGUua3RzIl0)